File: Without Node Module/Xampp/htdocs/core/week-1/c-programs/programs/24.php
<?php
    $num1 = 36;
    $num2 = 48;

    function calculateGCD($a, $b)
    {
        while ($b != 0) {
            $temp = $b;
            $b = $a % $b;
            $a = $temp;
        }
        return $a;
    }

    $gcd = calculateGCD($num1, $num2);
    echo "GCD of $num1 and $num2 is: $gcd";
?>

File: One Task/Task 1/c programming/24.c
// GCD

#include <stdio.h>

int main() {
    int n1 = 36, n2 = 60, GCD;
    for(int i=1; i<=n1 && i<=n2; i++) {
        if(n1%i == 0 && n2%i == 0) {
            GCD = i;
        }
    }
    printf("GCD of %d and %d is %d", n1, n2, GCD);
    return 0;
}